content/webapp/pages/page.js

```diff
@@ -46,18 +46,36 @@ type Props = {|
   page: PageType,
   siblings: SiblingsGroup[],
   children: SiblingsGroup,
+  ordersInParents: any[],
 |};
+
 export class Page extends Component<Props> {
   static getInitialProps = async (ctx: Context) => {
     const { id, memoizedPrismic } = ctx.query;
     const page = await getPage(ctx.req, id, memoizedPrismic);
     if (page) {
       const siblings = await getPageSiblings(page, ctx.req, memoizedPrismic);
+      const ordersInParents =
+        page.parentPages.map<{
+          parentId: string,
+          title: string,
+          order: number,
+          parentType: 'pages' | 'exhibitions',
+        }>(p => {
+          return {
+            parentId: p.id,
+            title: p.title,
+            order: p.order,
+            parentType: p.parentType,
+          };
+        }) || [];
```

Comment:
Can you explain what this is doing as I can't tell what this type is for?

Are we just appending the order to the parent page?

Reply:
> Are we just appending the order to the parent page?

Yes. And since a page can theoretically have multiple parents each potentially with a different order, it's necessarily an array.